ddtrace/contrib/openai/patch.py

```diff
@@ -0,0 +1,205 @@
+from ddtrace import config
+from ddtrace.contrib.trace_utils import set_flattened_tags
+from ddtrace.internal.agent import get_stats_url
+from ddtrace.internal.constants import COMPONENT
+from ddtrace.internal.dogstatsd import get_dogstatsd_client
+
+from .. import trace_utils
+from .. import trace_utils_async
+from ...pin import Pin
+from ..trace_utils import wrap
+from ._utils import append_tag_prefixes
+from ._utils import engine_resource_name
+from ._utils import process_request
+from ._utils import process_response
+from ._utils import process_text
+
+
+config._add(
+    "openai",
+    {
+        "_default_service": "openai",
+    },
+)
+
+REQUEST_TAG_PREFIX = "request"
+RESPONSE_TAG_PREFIX = "response"
+ERROR_TAG_PREFIX = "error"
+ENGINE = "engine"
```

Comment:
I think these could be moved into a constants file, i.e. `ddtrace.contrib.openai.constants.py`